### DISCUSSION TITLE
Timeline index

### DISCUSSION THREAD

**Discussion by M-ir-sta** [google/timesketch#1728] [Q&A, quick solutions, support] [answered]
Is there any way to get a list of all existing index of timelines using tsctl?
Thanks!

**Comment by kiddinn** [comment, marked as answer]
so are you looking for just the ES indices, a full list of them in a given instance? ATM it is not possible, but the API client has

```python
ts_client.list_searchindices()
```

 which will give you a list of all search indices that the user that is executing the query has access to.

**Reply by kiddinn** [reply]
this list will be a list of `SearchIndex` objects that then have other properties, such as...

```python
[(s.index_name, s.has_timeline_id, s.description) for s in ts_client.list_searchindices()]
```

**Comment by M-ir-sta** [comment]
It will be maybe a good idea to have an "administrative page on the GUI" to find these.

**Reply by kiddinn** [reply]
are you looking for a list of all indices within a single timeline or all indices that have been stored in Timesketch in general?

We are also adding in grafana/prometheus containers (they are already inside the dev container, will be released shortly into the release one as well).

You can get a quick overview of all these things there, utilization, number of indices, etc... you can try that already if you use the dev docker config, visit http://localhost:3000
